tools/destop-app/build.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
 # Build OpenMapper Desktop App
+
+cd wrapper/desktop/
 mkdir -p da_build
-cd build
-cmake ../wrapper/desktop-app  -DCMAKE_BUILD_TYPE=DEBUG 
-make -j8
+cd da_build
+cmake .. -DCMAKE_BUILD_TYPE=DEBUG 
+make -j4
```

Comment:
I just copied from the ros version, there the bin and build dir have to be near the CmakeLists and package.xml in order to be found... I think, we do not need this desktop-app in the future anyhow if we do that in the OpenMapperDesktop package